packages/client-python/src/rocketride/mixins/store.py

```
@@ -163,6 +162,22 @@ async def fs_mkdir(self, path: str) -> None:
         response = await self.request(request)
         self._check_response(response)
 
+    async def fs_rmdir(self, path: str, recursive: bool = False) -> None:
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)